### PR TITLE
Fixed Paths & minor bugs

### PR DESCRIPTION
**Pull request by ShrutiLakhani**

### CODE REVIEW

**Comment by aiman-ansari** [comment]
can you put your ecommerce link

File: Cart/Cart.html [hunk 1]
```diff
@@ -34,21 +34,21 @@
...
                     </div>
                     <div class="icon-ecomm">
-                        <span class="material-icons">
+                        <a href="/Cart/Cart.html"><span class="material-icons">
                             shopping_bag
-                        </span>
+                        </span></a>
                     </div>
             </nav>
                 </div>
```
Comment (aiman-ansari) [inline Cart/Cart.html:45]: You can use prettier or some other extension for proper indentation here

**Review by aiman-ansari — APPROVED** [review]
Everything is good to  merge

**Review by ashishkatiyar2404 — APPROVED** [review]
Your code looks good to me. UI also looks clean 
you can merge.💯

**Pull request merged**